src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -48,15 +48,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
 
             var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
             var memberNodeSymbolPairs = selectedMemberNodes
-                .SelectAsArray(m => (node: m, symbol: semanticModel.GetRequiredDeclaredSymbol(m, cancellationToken)))
-                .WhereAsArray(pair => MemberAndDestinationValidator.IsMemberValid(pair.symbol));
+                .SelectAsArray(m => (node: m, symbol: semanticModel.GetDeclaredSymbol(m, cancellationToken)))
+                .WhereAsArray(pair => pair.symbol != null && MemberAndDestinationValidator.IsMemberValid(pair.symbol));
```

Comment:
I don't feel like `pair.symbol != null` is needed.

`IsMemberValid` should already return `false` for null. You need to annotate its parameter as nullable to avoid warnings.


https://github.com/dotnet/roslyn/blob/eee708de830969a1bf16960b559e31c794297140/src/Features/Core/Portable/PullMemberUp/MemberAndDestinationValidator.cs#L30-L42